tests/ops/test_qubit_ops.py

```diff
@@ -688,6 +688,62 @@ def test_ISWAP_decomposition(self, tol):
 
         assert np.allclose(decomposed_matrix, op.matrix, atol=tol, rtol=0)
 
+    def test_isingxx_decomposition(self, tol):
+        """Tests that the decomposition of the IsingXX gate is correct"""
```

Comment:
How come these tests were missed in the original PR?

Reply:
They were not part of the original issue descriptions explicitly and so other tests were added. Wouldn't have liked to drag the review process further as the original PRs for unitaryHACK were in good shape otherwise.

Reply:
got it :+1: